Sources/Build/Triple.swift

```diff
@@ -48,11 +48,13 @@ public struct Triple {
         case darwin
         case macOS = "macosx"
         case linux
+        case windows
 
         fileprivate static let allKnown:[OS] = [
             .darwin,
             .macOS,
-            .linux
+            .linux,
+            .windows
```

Comment:
Could this use `CaseIterable` and `allCases` from [SE-0194](https://github.com/apple/swift-evolution/blob/master/proposals/0194-derived-collection-of-enum-cases.md)?

Reply:
It definitely could, but I feel like that belongs in a separate PR.